src/components/Header.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from '../styles/Header.module.css';
+
+const Header = () => {
+  const headerStyle = {
+    padding: '20px 0',
+    lineHeight: '1.5em',
+    color: '#aeadad',
+    textAlign: 'center',
+  };
```

Comment:
If you want to add `styles` like an `object` as you did, It would be better to add those `objects `outside your main `function`.
Example: You can add this `code` in line 3 in this case.
It would make your code more organized 🥰.